priam/src/main/java/com/netflix/priam/backupv2/SnapshotMetaTask.java

```diff
@@ -285,7 +287,26 @@ private void uploadAllFiles(final File backupDir) throws Exception {
                 // We do not want to wait for completion and we just want to add them to queue. This
                 // is to ensure that next run happens on time.
                 AbstractBackupPath.BackupFileType type = AbstractBackupPath.BackupFileType.SST_V2;
-                backupHelper.uploadAndDeleteAllFiles(snapshotDirectory, type, target, true);
+                backupHelper
+                        .uploadAndDeleteAllFiles(snapshotDirectory, type, target, true)
+                        .forEach(
+                                future ->
+                                        Futures.addCallback(
```

Comment:
not the biggest fan of this sparse formatting, but that's what `googleJavaFormat` did